lib/assets/SnapshotHelper.swift

```diff
@@ -16,6 +16,17 @@ func setLanguage(app: XCUIApplication)
     Snapshot.setLanguage(app)
 }
 
+func setLaunchArguments(app: XCUIApplication)
+{
+    Snapshot.setLaunchArguments(app)
+}
+
+func setupSnapshot(app: XCUIApplication)
```

Comment:
Added a general setup function to SnapshotHelper here, this will make it easier to add new functionality. If you like this @KrauseFx, I can update the template and readme too